test/commitVotes.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-env mocha */
+/* global contract assert */
+
+const utils = require('./utils.js');
+
+contract('PLCRVoting', (accounts) => {
+  describe('Function: commitVotes', () => {
+    const [alice, bob] = accounts;
+
+    it('should commit an array of 1 vote for 1 poll', async () => {
+      const plcr = await utils.getPLCRInstance();
+
+      const options = utils.defaultOptions();
+      options.actor = alice;
+
+      // start a poll
+      const receipt = await utils.as(options.actor, plcr.startPoll, options.quorum,
+        options.commitPeriod, options.revealPeriod);
+      const pollID = utils.getPollIDFromReceipt(receipt);
+
+      // verify that the commit period is active
+      const isActive = await plcr.commitPeriodActive(pollID);
+      assert(isActive, 'poll\'s commit period is not active');
+
+      // commit an array of 1 vote
+      const secretHash = utils.createVoteHash(options.vote, options.salt);
+      const prevPollID =
+        await plcr.getInsertPointForNumTokens.call(options.actor, options.numTokens, pollID);
+
+      const pollIDs = [pollID];
+      const secretHashes = [secretHash];
+      const numsTokens = [options.numTokens];
+      const prevPollIDs = [prevPollID];
+      try {
```

Comment:
Arguably we don't need to try/catch this, since we want to blow up anyway. We do get an error message out of it though. You use try/catches in this commitVotes file, but not in revealVotes. Any reason for that? If not, choose one style and be consistent.